setup.cfg

```diff
@@ -7,8 +7,13 @@ cover-min-percentage=100
 cover-erase=true
 
 [flake8]
+exclude=./docs/conf.py,./test/*,./urllib3/packages/*
 max-line-length=99
 
+[pep8]
```

Comment:
So now any time we update one of these stanzas we have to update the other. How convenient